core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -727,6 +732,27 @@ class ReplicaManager(val config: KafkaConfig,
     }
   }
 
+  def getPartitionOrException(topicIdPartition: TopicIdPartition): Partition = {
```

Comment:
The original name of this method that accept `topicPartition` is `getPartitionOrException` I feel it would be weird to name this `partitionOrException`